agent/nmap_agent.py

```diff
@@ -74,6 +74,15 @@ def __init__(
         self._scope_domain_regex: Optional[str] = self.args.get("scope_domain_regex")
         self._vpn_config: Optional[str] = self.args.get("vpn_config")
         self._dns_config: Optional[str] = self.args.get("dns_config")
+        self._ipv6_cidr_limit: int = int(
+            self.args.get("ipv6_cidr_limit", IPV6_CIDR_LIMIT)
+        )
+
+    def _validate_ipv6_settings(self) -> None:
+        """Validate IPv6-specific settings."""
+        max_mask = int(self.args.get("max_network_mask_ipv6", "128"))
```

Comment:
This will `int(self.args.get("max_network_mask_ipv6", "128"))` crash if the value of exists and it's None
max_network_mask_ipv6.

![image](https://github.com/user-attachments/assets/2ffb7fa7-d2de-4eec-85af-80bc5b2f4169)

Use int(data.get("max_network_mask_ipv6") or "128") instaed.

Reply:
what data?